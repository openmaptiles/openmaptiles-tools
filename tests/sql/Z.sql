-- Z
SELECT Z(1000000000) AS null_v1;	-- \\N
SELECT Z(500000000) AS v2;	-- 0
SELECT Z(1000) AS v3;	-- 19
SELECT Z(0) AS null_v4;	-- \\N
SELECT Z(NULL) AS null_v5;	-- \\N
