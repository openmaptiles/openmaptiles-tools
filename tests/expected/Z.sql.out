 null_v1 
---------
        
(1 row)

 v2 
----
  0
(1 row)

 v3 
----
 19
(1 row)

 null_v4 
---------
        
(1 row)

 null_v5 
---------
        
(1 row)

